.travis.yml

```diff
@@ -4,7 +4,7 @@ go:
   - 1.7
 
 install:
-  - ./install.sh
+  - go get -t ./...
```

Comment:
Let's keep it clean for now and say that if we want to do a R4M, we need to have crypto updated first...